### DISCUSSION TITLE
Ability to disable en-US keyboard layout from Win+Space layout switcher

### DISCUSSION THREAD

**Discussion by supra107** [valinet/ExplorerPatcher#1959] [Ideas]
I use a Polish keyboard layout and I also have a Russian mnemonic layout, however I also have to keep the English OS language active due to how Windows 10 deals with language. I was able to remove the en-US layout via registry tweaks, however ExplorerPatcher completely ignores it and always keeps the en-US layout enabled even when I remove English from Windows 10 entirely, or if I change it to use a different layout for that language.

I have no need for the en-US layout to be available since Polish uses Latin with additional accents, therefore it is useless for me and it's only an annoyance when I want to switch to Cyrillic since it has to be between Polish and Russian layouts. I would like to see an option in ExplorerPatcher to be able to exclude specific layouts from the layout selector or to simply hide the en-US one.

**Comment by S-ed** [comment]
I use Shift+Alt, and I have Japanese IME layout that has English keys. I don't need English layout. Please add this feature.

**Comment by Tabiskabis** [comment]
Sounds to me like an old bug that's been there since Win7. Try this:
Go to the "language options" of each of your display languages (listed in Language & region). Add your keyboard layout to all your display languages. When that's done, add the US keyboard layout, then remove it again. Do this with all display languages. The US layout should be gone now.
